integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/merge/MergeTestCase.scala

```diff
@@ -780,6 +780,68 @@ class MergeTestCase extends QueryTest with BeforeAndAfterAll {
       Seq(Row("c", "200"), Row("e", "100")))
   }
 
+  test("test new API") {
```

Comment:
change name to include more testcase description

Reply:
only one testcase added? Is there more test for upsert api?

Reply:
updated and added test cases